src/Form/CommentType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Comment;
+use App\Entity\Trick;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\NotBlank;
+
+class CommentType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('content', TextType::class, [
+                'constraints' => [
+                    new NotBlank(null, 'Ce champs ne doit pas être vide!', false)
```

Comment:
why twice ? once in the form, once in the entity

Reply:
I just forgot to delete it